apps/jobs/sender.py

```diff
@@ -55,27 +55,24 @@ def get_or_create_sqs_queue(queue_name):
     return queue
 
 
-def publish_submission_message(challenge_pk, phase_pk, submission_pk):
+def publish_submission_message(message):
     """
     Args:
-        challenge_pk: Challenge Id
-        phase_pk: Challenge Phase Id
-        submission_pk: Submission Id
+        message: A Dict with following keys
+            - "challenge_pk": int
+            - "phase_pk": int
+            - "submission_pk": int
+            - "submitted_image_uri": string, Only for Docker Based Challenge submissions
```

Comment:
str, (only available when the challenge is a code upload challenge)

Reply:
Done